test/modules/post/test/mssql.rb

```diff
@@ -41,6 +41,15 @@ def test_console_query
     end
   end
 
+  def test_datatypes
+    it "should support ntext TDS datatype" do
+      stdout = with_mocked_console(session) {|console| console.run_single(%{ query "select cast('foo' as ntext);"})}
```

Comment:
We'll probably want this to use the convention of the other PR, but can fix separately